cybench/models/trend_models.py

```diff
@@ -114,48 +113,56 @@ def _predict_trend(self, test_data: Iterable):
 
             sel_train_df = self._train_df[self._train_df[KEY_LOC] == loc]
             train_labels = sel_train_df[[KEY_YEAR, KEY_TARGET]].values
-            # trend_values = np.zeros(train_labels.shape)
             train_years = sorted(sel_train_df[KEY_YEAR].unique())
+            assert test_year not in train_years
 
```

Comment:
unrelated; but in pandas is not used in trend_models; yet it is imported in L3

Reply:
Removed